ios/ObjectPermissions/ToDo/WelcomeViewController.swift

```diff
@@ -0,0 +1,146 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2018 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import UIKit
+import RealmSwift
+
+// Configure the permissions on the Realm and each model class.
+// This will only succeed the first time that this code is executed. Subsequent attempts
+// will silently fail due to `canSetPermissions` having already been removed.
```

Comment:
I think the expectation is that code like this will not really be executed by the app in a real-life scenario, but rather the changes will be made by an admin through Studio or similar (given that it's a one-off thing for a system). Maybe add a remark about this in the comment so it doesn't end up becoming a too-common pattern?

Reply:
Is there an easy way to configure these permissions via Realm Studio at this point? It seems like it'd be pretty cumbersome to do via the normal object editing UI, even after you'd gone to the trouble of creating the permissions schema in the Realm.

Reply:
I'm going to add this comment here:

NOTE: The initial configuration of permissions would typically be done by an admin via Realm Studio or a dedicated script, not as part of your app's logic. We show it here solely to demonstrate a more advanced use of the Swift API.